lib/rubocop/cop/layout/space_around_method_call.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Layout
+      # Checks dot method calls to not have spaces around them.
+      #
+      # @example
+      #   # bad
+      #   foo. bar
+      #   foo .bar
+      #   foo . bar
+      #   foo. bar .buzz
+      #   foo
+      #     . bar
+      #     . buzz
+      #
+      #   # good
+      #   foo.bar
+      #   foo.bar.buzz
+      #   foo
```

Comment:
You can also have chaining in a suffix form.

Reply:
Chaining in a suffix form is a bad style. There is already a cop for that so not including it in the example. 
I have added a spec for chaining in suffix form though.